.claude/specs/resolve-graph-interrupt-error/PROPOSAL_GUARDIAN.md

```
@@ -0,0 +1,771 @@
+    })
+    raise
+except AssistantConfigurationError as e:
+    # Handle corrupted/invalid config
+    ...
+```
+
+### 3.2 Agent Construction Failure
+
+**What if the agent construction fails?**
+
+**Current Risk**: `construct_agent()` (flows/__init__.py line 240-242) catches all exceptions and re-raises:
```

markdownlint-cli2 (0.18.1)

Strong style
Expected: asterisk; Actual: underscore

(MD050, strong-style)